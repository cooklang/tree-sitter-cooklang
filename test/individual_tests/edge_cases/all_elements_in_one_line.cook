>> servings: 4
= Section Name
-- Comment here
> Note here
In #pan{} add @oil{2%tbsp} and heat ~{3%min} [- block comment -] then serve.