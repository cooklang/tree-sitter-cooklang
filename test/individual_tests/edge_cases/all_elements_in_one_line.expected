(recipe
  (metadata)
  (section
    header: (section_header
      name: )
    content: (comment)
    content: (note)
    content: (step
      (plain_text)
      (cookware
        name: (cookware_text))
      (plain_text)
      (ingredient
        name: (ingredient_text)
        (amount
          (quantity)
          (units)))
      (plain_text)
      (timer
        (amount
          (quantity)
          (units)))
      (plain_text)
      (block_comment)
      (plain_text))))